test/ad/utils.jl

```diff
@@ -105,58 +105,51 @@ function test_ad(dist::DistSpec; kwargs...)
         end
     end
 
-    if isempty(θ)
-        # In this case we can only test the gradient with respect to `x`
-        xtest = vectorize(x)
-        ftest = let xorig=x
-            x -> f_allargs(unpack(x, (1,), xorig)...)
-        end
-        test_ad(ftest, xtest; kwargs...)
-    else
-        # For all combinations of distribution parameters `θ`
-        for inds in combinations(2:(length(θ) + 1))
-            # Test only distribution parameters
+    # For all combinations of distribution parameters `θ`
+    for inds in powerset(2:(length(θ) + 1))
+        # Test only distribution parameters
+        if !isempty(inds)
             xtest = mapreduce(vcat, inds) do i
                 vectorize(θ[i - 1])
             end
-            ftest = let xorig=x, θorig=θ, inds=inds
-                x -> f_allargs(unpack(x, inds, xorig, θorig...)...)
+            ftest = let xorig = x, θorig = θ, inds = inds
+                x->f_allargs(unpack(x, inds, xorig, θorig...)...)
```

Comment:
I'm not sure if we have a style-guide for anonymous functions, but don't we usually do ` -> ` instead of `->`?